maven-javadoc-plugin/pom.xml

```diff
@@ -193,9 +193,9 @@ under the License.
 
     <!-- misc -->
     <dependency>
-      <groupId>commons-lang</groupId>
-      <artifactId>commons-lang</artifactId>
-      <version>2.6</version>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.6</version>
```

Comment:
The support for Java 6 is because the plugin can be used with all Maven 3.x versions. Unless there is a strong reason to require Java 7 (like a 3rd party dependency that has to be updated to a version requiring Java 7 in order to fix bugs), it'd be preferable to keep with this compability for now.